test/components/connect.spec.js

```diff
@@ -332,12 +332,20 @@ describe('React', () => {
       expect(decorated.state.data.testFunc).toBeA('function')
       expect(decorated.state.data.deferredFetch).toEqual(undefined)
 
-      decorated.state.data.testFunc('A', 'B')
+      decorated.state.data.testFunc('A', 'B').then(value => expect(value).toEqual({ T: 't' }))
```

Comment:
Good point about making sure `then` was actually called. if we decide to keep this i'll add that to the test